.travis.yml

```diff
@@ -43,9 +43,13 @@ matrix:
       script:
         - docker load -i $HOME/docker/images.tar || true
         - docker image list
-        - docker build --cache-from ${GOOGLE_DOCKER_IMAGE} -t ${IMAGE} .
+        - docker build --cache-from ${GOOGLE_DOCKER_IMAGE} -t ${IMAGE_BRANCH_COMMIT} .
+        - make gcloud-login
+        - IMAGE=${IMAGE_BRANCH_COMMIT}
         - make docker-push
-        - docker tag ${IMAGE} ${GOOGLE_DOCKER_IMAGE}
+        - make pubsub-publish
+        - docker tag ${IMAGE_BRANCH_COMMIT} ${GOOGLE_DOCKER_IMAGE}
```

Comment:
I guess the goal of this line is to create a `latest` tag; why not making it explicit?
Also, why not moving this logic into `ci/Makefile:deploy` repo as in [src-d/ci/Makefile.main](https://github.com/src-d/ci/blob/v1/Makefile.main#L224)?

If we do so, we could use the same `script` commands in all our projects.